### PR TITLE
Implements stylized 404 with google search link

### LINKED ISSUE #8: Improve 404 page
Add CSS, and google search option

### PR DESCRIPTION
**Pull request by anudaweerasinghe**
Resolves #8

287172145_1033147490928916_363761456851635884_n

### CODE REVIEW

File: commands/course.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 name: Course Tool
-description: Search info about courses at CMU like "fce 122"
+description: Search info about courses at CMU like "fce <c"
```
Comment (hdavidethan) [inline commands/course.yml]: What's the <c?

File: src/index.ts [hunk 2]
```diff
@@ -91,9 +91,9 @@ const readCommands = (): CommandMapping => {
     res.redirect(url.toString());
   });
 
-  const errorHandler: express.ErrorRequestHandler = (value: { token: string, closest: string, currQuery: string }, req, res, next) => {
+  const errorHandler: express.ErrorRequestHandler = (value: { token: string, closest: string, currQuery: string, currSearch:string }, req, res, next) => {
```
Comment (hdavidethan) [inline src/index.ts:94]: We should probably make this into a new type

**Review by hdavidethan — APPROVED** [review]
lgtm

File: src/index.ts (hunk 2 above)
Comment (hdavidethan) [reply src/index.ts:94]: I made this into a new issue #10

File: commands/course.yml (hunk 1 above)
Comment (anudaweerasinghe) [reply commands/course.yml]: oh typo

**Pull request merged**